client/app/account/billing/main/history/downloadBills/billing-main-history-downloadBills.html

```diff
@@ -0,0 +1,51 @@
+<oui-modal data-heading="{{:: 'billing_main_bill_download_title' | translate }}"
```

Comment:
Shouldn't it be a full screen page instead ?

Reply:
Hmm for the moment we agreed with @lizardK that a modal can fit the needs, since there is only three fields to search. 
(Plus the action is available into a dropdown menu)